### ISSUE TITLE
Where is the training data?

### ISSUE THREAD

**Issue by wzm2256** [chaneyddtt/UDA-Animal-Pose#7]
I follow the data processing instructions and download everything including the pretrained model and the fake labels and put them in the correct place. However, I can not start training because it requires the image `animal_data/synthetic_animal/horse_combineds5r5_texture/1000_SK_horse_skeleton_horse_idleB_anim_0.65_450.00_160.00_150.00.png_img.png`. However, my `synthetic_animal/horse_combineds5r5_texture` folder only contains 3 files which are  `mean.pth.tar` `train_idxs.npy` `valid_idxs.npy`. 

Any idea what is going on?

**Comment by wzm2256** [comment]
By the way, I have checked other sub-folders including `elephant`, `sheep`, `hound` in the same folder. These sub-folders are full of `npy` and `png` files, thus I believe my downloading process is correct.

**Comment by wzm2256** [comment]
I see what is going on. The data is not downloaded at all.

The two lines for downloading these two datasets are comment out in `get_dataset.sh`.

I'm confused. I thought the synthetic data is needed in both the pretrain and the transfer stage, right?

**Comment by chaneyddtt** [comment]
Hi @wzm2256, thanks for reminding. Yes, you need to comment or uncomment some lines to get the data you want.  And yes, you are correct, the synthetic data is needed in both the pretrain and the transfer stage.

**Comment by wzm2256** [comment]
Thanks for your reply! I will try to uncomment these lines.
